tests/job_tests/recognition/test_scoring.py

```diff
@@ -0,0 +1,91 @@
+import os
+import tempfile
+from typing import Optional
+
+from sisyphus import tk, setup_path
+
+from i6_core.tools.compile import MakeJob
+from i6_core.tools.git import CloneGitRepositoryJob
+from i6_core.recognition import ScliteJob
+
+Path = setup_path(__package__)
+
+
+def compile_sctk(
+    branch: Optional[str] = None,
+    commit: Optional[str] = None,
+    sctk_git_repository: str = "https://github.com/usnistgov/SCTK.git",
+) -> tk.Path:
+    """
+    :param branch: specify a specific branch
+    :param commit: specify a specific commit
+    :param sctk_git_repository: where to clone SCTK from, usually does not need to be altered
+    :return: SCTK binary folder
+    """
+    sctk_job = CloneGitRepositoryJob(url=sctk_git_repository, branch=branch, commit=commit)
+    sctk_job.run()
+
+    sctk_make = MakeJob(
+        folder=sctk_job.out_repository,
+        make_sequence=["config", "all", "check", "install", "doc"],
+        link_outputs={"bin": "bin/"},
+    )
+    sctk_make.run()
+    # This is needed for the compilation to work in the i6 environment, otherwise still untested
+    sctk_make._sis_environment.set("CPPFLAGS", "-std=c++11")
+    return sctk_make.out_links["bin"]
+
+
+def test_sclite_job():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        from sisyphus import gs
+
+        gs.WORK_DIR = tmpdir
+        sctk_binary = compile_sctk(branch="v2.4.12")
+        hyp = Path("files/hyp.ctm")
+        ref = Path("files/ref.stm")
+
+        sclite_job = ScliteJob(ref=ref, hyp=hyp, sctk_binary_path=sctk_binary)
+        sclite_job._sis_setup_directory()
+        sclite_job.run()
+
+        assert sclite_job.out_wer.get() == 58.8, "Wrong WER, %s instead of 58.8" % str(sclite_job.out_wer.get())

```

Comment:
would you also add a test for a non-default precision?

Reply:
This test is supposed to be applied before #442.